AudioKit/Common/Internals/AudioKitHelpers.swift

```diff
@@ -184,8 +188,8 @@ extension Double {
             // exponential taper
             var adjustedMinimum: Double = 0.0
             var adjustedMaximum: Double = 0.0
-            if range.lowerBound == 0 { adjustedMinimum = 0.000_000_000_01 }
-            if range.upperBound == 0 { adjustedMaximum = 0.000_000_000_01 }
+            if range.lowerBound == 0 { adjustedMinimum = 0.00_000_000_001 }
```

Comment:
Why this change? Seems wrong to me.

Reply:
oh, the swift lint did that. feel free to revert.

Reply:
I actually realized that it was correct this way, which is why I accepted the PR as is rather than ask for a change.